### PR TITLE
new test and more stable achiever

### PR DESCRIPTION
**Pull request by DaddyWesker**
Some changes that makes achiever to work more stable. And tests for blockdrops PR in Vereya.

### CODE REVIEW

**Comment by Necr0x0Der** [comment]
It doesn't seem to be working for me. When I ask the agent to create a stone_pickaxe, it collects planks and sticks, but doesn't create a wooden_pickaxe and tries to mine stone with sticks. Without this PR, the agent works. It collects one more log, creates and uses wooden_pickaxe

**Comment by DaddyWesker** [comment]
That is weird. I'll try to reproduce it.

File: examples/minelogy.py [hunk 1]
```diff
@@ -12,95 +14,88 @@ def __init__(self, item_list, items_to_craft, mcrecipes):
...
-                   'tools': [None]},
-                  {'type': 'stick'}
-                  ),
-                 ({'blocks': [{'type': 'tallgrass'}],
-                   'tools': [None]},
-                  {'type': 'wheat_seeds'}
-                  ),
-                 ]
+        self.mines = []
+        # self.mines = [({'blocks': [{'type': 'log'}],
```
Comment (noskill) [inline examples/minelogy.py:18]: i would rather keep only a small part of this code block. A couple of tool-block examples would be enough for documentation purposes.

Comment (DaddyWesker) [reply examples/minelogy.py]: @noskill  done

**Pull request merged**